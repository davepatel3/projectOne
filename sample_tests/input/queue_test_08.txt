int
2
2
4
pf
gb
pb
ef 38
gc
gb
eb 75
df
db
ef 12
ps
eb -27
ps
eb 24
ef 43
ef -45
eb 89
ef -35
eb -98
ef 42
df
ps
gs
ef 89
db
ef -38
gs
pf
pb
df
gb
ps
eb 64
gc
gc
pf
eb 85
eb -57
pf
eb -44
pb
db
ef -76
ef -15
qt
