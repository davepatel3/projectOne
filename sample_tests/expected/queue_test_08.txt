cyclic_double_queue is empty!
cyclic_double_queue is empty!
cyclic_double_queue is empty!
2
38
38
75
[-][+]
[+][+]
42
[+][+][+][+][+][+][+][+][-][-][-][-][-][-][-][-]
8
-98
9
89
24
[+][+][+][+][+][+][-][-][-][-][-][-][-][-][-][-]
16
16
-57
